src/openeo_gfmap/utils/catalogue.py

```diff
@@ -107,19 +133,20 @@ def _check_cdse_catalogue(
     return len(grd_tiles) > 0
 
 
-def s1_area_per_orbitstate(
+def s1_area_per_orbitstate_vvvh(
     backend: BackendContext,
     spatial_extent: SpatialContext,
     temporal_extent: TemporalContext,
 ) -> dict:
     """Evaluates for both the ascending and descending state orbits the area of interesection
     between the given spatio-temporal context and the products available in the backend's
-    catalogue.
+    catalogue. Only works for the products with the VV&VH polarisation.
```

Comment:
Evaluates for both the ascending and descending state orbits the area of interesection for the VV&VH polarisation